brave/browser/api/brave_api_extension.cc

```diff
@@ -98,10 +107,17 @@ scoped_refptr<extensions::Extension> LoadExtension(const base::FilePath& path,
     return NULL;
 
   std::vector<extensions::InstallWarning> warnings;
-  if (!extensions::file_util::ValidateExtension(extension.get(),
-                                                error,
-                                                &warnings))
-    return NULL;
+
+  int resource_id;
+  if (!IsComponentExtension(path, &resource_id)) {
```

Comment:
can we use IsComponentExtensionResource here instead? `IsComponentExtension` is a bit misleading because we have component extensions that are not loaded from resource paks